app.js

```diff
@@ -335,16 +335,28 @@ aws.initConfig(awsProps)
                     const id = req.params.id;
                     storageHandler.expandId(id)
                         .then(result => {
+                            const config = JSON.parse(result.config);
                             const metadata = {
-                                ogDescription: result.metadata ? result.metadata.description.S : null,
-                                ogAuthor: result.metadata ? result.metadata.author.S : null,
-                                ogTitle: result.metadata ? result.metadata.title.S : null
+                                ogDescription: result.special_metadata ? result.special_metadata.description.S : null,
```

Comment:
Is there a reason we're using snake_case instead of camelCase here? (does it match with something external?) I suspect it matches something in the S3 document, right?